examples/templates/aws-ecs-container/main.tf

```diff
@@ -92,15 +92,19 @@ resource "aws_ecs_service" "workspace" {
 data "coder_workspace" "me" {}
 
 resource "coder_agent" "coder" {
-  arch           = "amd64"
-  auth           = "token"
-  os             = "linux"
-  dir            = "/home/coder"
-  startup_script = <<EOT
-    #!/bin/bash
```

Comment:
Removed all shebangs because I didn't want users to get the wrong idea that we obey the shebang. In unix, we should run script via `/bin/bash` if it has this header, but we don't. Likewise for `#!/bin/sh` we should use `/bin/sh`, but we don't.